eden/scm/edenscm/ext/github/gh_submit.py

```diff
@@ -88,6 +89,10 @@ async def get_repository(
         upstream = None
     return _parse_repository_from_dict(repo, hostname=hostname, upstream=upstream)
 
+class PullRequestState(enum.Enum):
```

Comment:
I realize that I have let this PR lapse and I have had to rebase it a bunch of times because there have been many changes in this area, but now that I take a deeper look, these enum values do not seem right. From https://docs.github.com/en/graphql/reference/enums#pullrequeststate the three values are:

- `CLOSED`
- `MERGED`
- `OPEN`

Where did `ALL` come from here?

Reply:
Hey @bolinfest! Sorry somehow I never got notified of your comment here. I honestly don't remember where `ALL` came from 😂 `CLOSED` `MERGED` `OPEN` looks correct. 

I guess this change made it into Sapling in https://github.com/facebook/sapling/commit/117e4dca50e8a58c64904a79dfa812ac07e5c6fb?